run-ios-flavor.bash

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Sorry for missing this in the first pass, but can you rename this file to run-ios-flavor.sh? ".bash" is an odd suffix. (also remember to update the name in the documentation)